docs/reference/query-languages/esql/esql-commands.md

```diff
@@ -31,6 +31,7 @@ An {{esql}} source command produces a table, typically with data from {{es}}. An
 
 {{esql}} supports these processing commands:
 
+* [preview] [`CHANGE_POINT`](#esql-change-point)
```

Comment:
To match other commands with `_` characters, this link should be `esql-change_point` (see `esql-mv_expand` for example).